includes/tasks/class-ss-setup-task.php

```diff
@@ -42,37 +46,29 @@ public function perform() {
 		// ->delete_all();
 
 		// add origin url and additional urls/files to database
-		self::add_origin_and_additional_urls_to_db( $this->options->get( 'additional_urls' ) );
+        self::add_page_from_url($origin_url, "Origin URL");
+
+        $additional_urls = $this->options->get( 'additional_urls' );
+        $additional_urls = array_unique( Util::string_to_array( $additional_urls ) );
+        foreach ($additional_urls as $url) {
```

Comment:
Moving this loop inside `perform` allows us to make a single generic method for adding URLs.